src/Utility/cleanup.lua

```diff
@@ -43,7 +43,15 @@ local function cleanup(task: any)
 		elseif typeof(task.Destroy) == "function" then
 			task:Destroy()
 
-		-- case 6: array of tasks
+		-- case 6: disconnect() function
+		elseif typeof(task.disconnect) == "function" then
+			task:disconnect()
+
+		-- case 7: Disconnect() function
+		elseif typeof(task.Disconnect) == "function" then
+			task:Disconnect()
+
+		-- case 8: array of tasks
```

Comment:
I feel like the allowed cases here will need some more thought

Reply:
It's worth mentioning that `cleanup` is used in other places than just `[Cleanup]` - for example, it's used as the default destructor for `ComputedPairs`. This is a breaking change, and not strictly related to this PR, so please break this out into it's own PR if you'd like to see this changed :)

Reply:
That's a fair point, I was just considering the idea that you might want to connect custom signal connections (Not RBXScriptConnection) to the cleanup utility. But you're right, it's not strictly related to this PR so I'll go ahead and remove it from this PR.

Since this doesn't address an existing issue or enhancement it might be better to create a new issue for it, rather than a new PR. Thoughts?

Reply:
That sounds like the right move to me :)